ens/ens.py

```diff
@@ -1,3 +1,4 @@
+# flake8: noqa: B008
```

Comment:
See my comment on `async_ens.py` we should address the issues instead, set to `None`, etc...